src/services/resource.js

```diff
@@ -1289,13 +1291,24 @@ module.exports = class resourceHelper {
 				}
 			)
 
-			if (resource === 0) {
+			if (count === 0) {
 				return responses.failureResponse({
 					message: 'RESOURCE_NOT_FOUND',
 					statusCode: httpStatusCode.bad_request,
 					responseCode: 'CLIENT_ERROR',
 				})
 			}
+
+			if (parentId && version != common.DEFAULT_RESOURCE_VERSION) {
```

Comment:
@adithyadinesh0412 Can we check the blobpath and version update in the min approvals is reached ? like in the review flow, can  u make sure this is working in self as well, there the blob path should be updated of the parent when it published via review (min approval reached)